be/src/agent/pusher.cpp

```diff
@@ -47,16 +47,15 @@ Pusher::Pusher(OLAPEngine* engine, const TPushReq& push_req)
 
 Pusher::~Pusher() {}
 
-AgentStatus Pusher::init() {
-    AgentStatus status = DORIS_SUCCESS;
+Status Pusher::init() {
+    Status status = Status::OK();
```

Comment:
Remove this line and define it at line 74

Reply:
I already removed pusher.cpp file in next pr.